src/sentry/static/sentry/app/views/organizationStream/overview.jsx

```diff
@@ -125,13 +131,30 @@ const OrganizationStream = createReactClass({
         this._poller.disable();
       }
     }
-    if (prevProps.params.searchId != this.props.params.searchId) {
-      this.onSavedSearchChange();
-    } else if (
-      prevProps.location.search != this.props.location.search ||
-      !isEqual(prevProps.selection, this.props.selection)
+
+    // If project selections have changed we need to get new processing issues.
+    if (!isEqual(prevProps.selection.projects, this.props.selection.projects)) {
+      this.fetchProcessingIssues();
+    }
+
+    let prevQuery = prevProps.location.query;
+    let newQuery = this.props.location.query;
+
+    // If any important url parameter changed or saved search changed
+    // reload data.
+    if (
+      !isEqual(prevProps.selection, this.props.selection) ||
+      prevQuery.sort != newQuery.sort ||
+      prevQuery.query != newQuery.query ||
```

Comment:
Is there a reason for non strict equality checks here vs strict below?

Reply:
No. Just an oversight. I'll fix it up now.